resources/lib/modules/system.py

```diff
@@ -612,12 +612,16 @@ def do_send_logs(self, log_cmd):
         result = os_tools.execute(log_cmd, get_result=True)
         if not paste_dlg.iscanceled():
             paste_dlg.close()
-            link = result.find('http')
-            if link != -1:
-                log.log(result[link:], log.WARNING)
-                xbmcDialog.ok('Paste complete', f'Log files pasted to {result[link:]}')
+            if result:
+                link = result.find('http')
+                # TODO Localization
+                if link != -1:
+                    log.log(result[link:], log.WARNING)
+                    xbmcDialog.ok('Paste complete', f'Log files pasted to {result[link:]}')
+                else:
+                    xbmcDialog.ok('Failed paste', 'Failed to paste log files. Try again')
             else:
-                xbmcDialog.ok('Failed paste', 'Failed to paste log files, try again')
+                xbmcDialog.ok('Failed paste', 'Failed to paste log files. Try again.')
```

Comment:
Let's be polite, i.e. "Please try again."

Reply:
Done.